src/docfx/lib/HrefUtility.cs

```diff
@@ -66,6 +66,10 @@ public static bool IsAbsoluteHref(string str)
                 || Uri.TryCreate(str, UriKind.Absolute, out _);
         }
 
+        public static bool IsHttpHref(string str)
+            => !string.IsNullOrEmpty(str) && Uri.TryCreate(str, UriKind.Absolute, out var uriResult)
```

Comment:
nit: `Startswith("http://")  || StartsWith("https://")` ?

is "https://///a.com" considered http?

Reply:
StartWith will not work for your case. https://///a.com is not http

---
In reply to: [203931273](https://github.com/dotnet/docfx/pull/3105#discussion_r203931273) [](ancestors = 203931273)

Reply:
will add test for your case

---
In reply to: [203931759](https://github.com/dotnet/docfx/pull/3105#discussion_r203931759) [](ancestors = 203931759,203931273)